electron/app/components/CheckboxGrid.tsx

```diff
@@ -12,24 +12,35 @@ const CHECKBOX_PADDING = 3;
 const CHECKBOX_TOTAL_SIZE = CHECKBOX_SIZE + 2 * CHECKBOX_PADDING;
 const LABEL_PADDING_RIGHT = 6;
 
-const Body = styled.div`
+export const Body = styled.div`
   vertical-align: middle;
 
   label {
     width: 100%;
+    height: 32px;
     margin-top: 3px;
     margin-bottom: 3px;
     margin-left: 0;
     margin-right: 0;
+    padding: 0.2em;
     border-radius: 2px;
+    display: flex;
+    justify-content: space-between;
 
     .MuiTypography-body1 {
+      flex: 1;
       font-size: unset;
       align-items: center;
+      padding-right: 4px;
+      max-width: 100%;
+    }
+
+    .MuiTypography-body1.with-checkbox {
+      max-width: calc(100% - 24px);
     }
 
     .MuiCheckbox-root {
-      padding: ${CHECKBOX_PADDING}px;
```

Comment:
I thought the constants were useful - is there a reason you stopped using them?
On second glance, it looks like you've repurposed this Body component to also be used as the Container component of SampleModal - is there a reason you couldn't use CheckboxGrid there where you needed it?

(If you're not using the constants anymore, they could be deleted above - a bit hard for me to tell from the diff if they're all still used)

Reply:
Excluding icons, I think we should avoid material ui (or any component framework) at all costs. Granted, I just used it for the confidence slider.

My recommendation is that we use their `classes` prop moving forward. https://material-ui.com/customization/components/#overriding-styles-with-classes

The constants are useful, but I also think we should use `rem` as much as possible because it is a derived unit.

Reply:
I would prefer `em` over `rem` in most cases since it's relative to the current element's font size. But in this case, the units were relative to an icon styled as `24px x 24px` (or maybe 18), which is why I went with `px` (plus I didn't want it to depend on font size). Granted, it's a minor change, and I think it looks just as good now, so I don't think changing it back is necessary. But it would be good to remove the constants if they're unused.

If using `classes` lets us avoid depending on the `Mui-*` class names, I'm entirely in favor of that. One issue I'd look out for is that some MUI components consist of multiple nested elements, and in a couple cases, I needed to style an internal element that may be difficult to add a custom class to with `classes`.

Reply:
Yes, I will address (forgot to mention that). Agree with the points above. I do believe that mui exposes everything that one may need to tweak. I'm still learning their patterns too, of course.